consumer/dvo_rules_consumer_test.go

```diff
@@ -421,21 +417,7 @@ func TestProcessingEmptyMetricsMissingAttributesWithClosedStorage(t *testing.T)
 	helpers.FailOnError(t, err, "empty report should not be considered an error at HandleMessage level")
 }
 
-func TestProcessingValidDVOMessageEmptyReportWithRequiredAttributesWithClosedStorage(t *testing.T) {
```

Comment:
did you remove this test on purpose?

Reply:
Yep because if the report is empty we don't touch the DB. I copied this test from the OCP storage but it makes no sense. I realised later :) The behaviour should be tested in `TestProcessingEmptyMetricsMissingAttributesWithClosedStorage`